api/database/factories/UserMocks.php

```diff
@@ -70,3 +70,16 @@
         'is_default' => $faker->boolean(),
     ];
 });
+
+$factory->define(\Spira\Rate\Model\Rating::class, function (\Faker\Generator $faker) {
+    return [
+        'rating_id' => $faker->uuid(),
+        'rating_value' => $faker->numberBetween(1, 10),
```

Comment:
Tinyint has 256 possible values? So max 255 if unsigned.
On `rateable_id` & `rateable_type` I realise that, but for the documentation it is better to have more realistic values returned. It shouldn't affect the tests.